### DISCUSSION TITLE
Add support for media source integration.

### DISCUSSION THREAD

**Discussion by niemyjski** [home-assistant/frontend#8001] [Accessibility]
I'd love to have a picture or media card that could show a picture or video based on a path. I'd use this to show what mail is arriving today (`/media/mail/mail_today.gif`) or who last opened my mailbox (`/media/mail/last_opened.jpg`) as described [here](https://blakeniemyjski.com/blog/how-to-get-notifications-when-your-mailbox-is-opened/). This doesn't work with the current picture card because of authentication.

**Comment by FrnchFrgg** [comment]
I would very much welcome this: having lovelace send the required auth token from lovelace cards. Maybe by adding support for the media-source:// URI scheme (as already exists for service calls according to the documentation in https://www.home-assistant.io/integrations/media_source )
